### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by ashmalzahra**
- Console app should behave in the same way as it is in the [example](https://www.youtube.com/watch?v=vkkgrhD6aXQ).
- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      parent_permission = false
+    else
+      puts "Invalid input. Please enter 'Y' or 'N'"
+    end
+
+    student = Student.new(age, name, parent_permission: parent_permission)
+    @people << student
+
+    puts 'Student created successfully'
+  end
```
Comment (Gambit142) [inline app.rb]: - [ ] When I try to create a student through the UI, I get this error:

image

#### The way you are creating the new instance of the Student class is not correct. The only keyword argument available in the Student class during initialization is the classroom parameter and it has a default value. The correct way to create a new instance of the Student class would be:
` student = Student.new(age, name, parent_permission)`. Kindly implement this change. 😎 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @ashmalzahra 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Created UI based on the video provided ✔️ 
- Application entry point is the `main.rb` file ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.rb (hunk 1 above)
Comment (ashmalzahra) [reply app.rb]: Hello, @Gambit142 Thank you for pointing this out. Although, if I create the Student in my console, it works perfectly fine, as shown in the picture, but I've still implemented the requested changes and I'll further look into it!
library

**Review by prolajumokeoni — APPROVED** [review]
Hi @ashmalzahra 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - Changes requested have ben implemented ✅ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**